web_external/Featuresets/FeaturesetsView.js

```diff
@@ -24,6 +24,11 @@ isic.views.FeaturesetsView = isic.View.extend({
         }, this);
         this.featuresets.fetch();
 
+        // TODO: Use the more general 'update' event, once Backbone is upgraded from Girder
+        this.listenTo(this.featuresets, 'remove', function () {
+            this.render();
+        }, this);
```

Comment:
Yep, I tried to mention the error in the line above...above, but didn't communicate that well!